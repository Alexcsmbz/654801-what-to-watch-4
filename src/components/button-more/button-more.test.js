import ButtonMore from './button-more.jsx';

const className = `catalog__button`;
const name = `Show more`;

describe(`ButtonMore snapshot test`, () => {
  it(`ButtonMore should render button`, () => {
    const tree = renderer.create(
        <ButtonMore
          onClick={() => { }}
          className={className}
          name={name}
        />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
