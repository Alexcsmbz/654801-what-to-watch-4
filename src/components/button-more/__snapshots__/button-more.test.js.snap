// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ButtonMore snapshot test ButtonMore should render button 1`] = `
<button
  className="catalog__button"
  onClick={[Function]}
  type="button"
>
  Show more
</button>
`;
